CONTRIB-234-392: Translate guidelines (#416)
feature-sliced/documentation: .github/pull_request_template.md

## CHANGELOG

<!-- 1. Describe the proposed changes in a couple of sentences/paragraphs -->

- Added ...smth...

> - Fix ...smth...

Reviewer: ```suggestion
- Fixed ...smth...
```
Для consistency

> <!-- Screenshots/Screencasts are welcome -->

Reviewer 2: [praise](https://conventionalcomments.org/#labels): Супер идея!

## Checklist

> <!-- 2. Before submitting changes for review, check the checklist and mark the necessary items at the end -->

Reviewer: `check the checklist` немного тавтологично, я бы предложил вариант `go over the checklist`

<!--

> Our common goal is to reduce review costs and achieve consistency in the code base. 🤙 

Reviewer: ```suggestion
Our common goal is to reduce review costs and achieve consistency in the codebase. 🤙 
```
Если я не ошибаюсь, обычно пишут так

Reviewer: Хотя на википедии и так, и так

Let's appreciate each other's time =)
-->

> - [ ] I'm familiarized with **[CONTIBUTING guidelines](https://github.com/feature-sliced/documentation/blob/master/CONTRIBUTING.md)**

Reviewer: Я думаю, тут лучше использовать `I reviewed`, т.к. грамматически было бы вернее `I have become familiar` или `I familiarized myself`, но оно как-то не звучит в данном контексте

Reviewer: Или же `I am familiar`, но это больше про знание, чем действие

> - [ ] I conducted a **[self-review of your changes](https://blog.beanbaginc.com/2014/12/01/practicing-effective-self-review/)**

Reviewer: ```suggestion
- [ ] I conducted a **[self-review of the changes](https://blog.beanbaginc.com/2014/12/01/practicing-effective-self-review/)**
```
Думаю, можно просто без пренадлежности

## CHANGELOG

<!-- 1. Describe the proposed changes in a couple of sentences/paragraphs -->

- Added ...smth...
- Fix ...smth...

> <!-- Screenshots/Screencasts are welcome -->

Author: Мб хоть так больше будут скринов прикладывать...
